src/conduits/RwaOutputConduit3.sol

```diff
@@ -364,7 +364,7 @@ contract RwaOutputConduit3 {
                     Math
     //////////////////////////////////*/
 
-    uint256 internal constant WAD = 10**18;
+    uint256 internal constant WAD = 10**18; // TB - as discussed maker's convention is to put the MATH part in the beginning
```

Comment:
Since it's just a matter of preference, this is one of the coding standards that we established in CES.